### PR TITLE
More detailed image_url information

### PR DESCRIPTION
**Pull request by hosebei**
As Discussed in https://github.com/home-assistant/core/issues/90121, the usage of the image_url tends to be unclear. Added detailed information of the image_url string and requirements with link to teams cards documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/msteams.markdown [hunk 1]
```diff
@@ -50,6 +50,8 @@ The following attributes can be placed inside `data` for extended functionality.
 | ---------------------- | -------- | ----------- |
 | `image_url`            |      yes | Attach an image to the message.
 
+The image URL must start with http, and as outlined from Microsoft in the [Documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#common-properties-for-all-cards), the picture must be on a publicly available location.
```
Comment (frenck) [inline source/_integrations/msteams.markdown]: According to the documentation you referenced, it must be an HTTPS URL?

```suggestion
The image must be an HTTPS URL, and as outlined by Microsoft in the [Documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#common-properties-for-all-cards), the picture must be on a publicly available location.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Put this PR back into draft, as the requested change / suggestion wasn't applied or responded on. Did you forgot to push a commit maybe @hosebei?

../Frenck

**Comment by hosebei** [comment]
I'm very sorry, I thaught the buttons i pressed accepted you corrections and rerequested the merge.
I'm doing better.

**Review by frenck — APPROVED** [review]
Don't worry about it! 

Thanks for the PR @hosebei 👍 

../Frenck

**Pull request merged**